bedrock/firefox/views.py

```diff
@@ -692,3 +692,15 @@ class FirefoxHubView(BlogPostsView):
     blog_slugs = 'firefox'
     blog_tags = ['home']
     template_name = 'firefox/hub/home.html'
+
+
+def sync_page(request):
+    locale = l10n_utils.get_locale(request)
+
+    if lang_file_is_active('firefox/sync', locale):
```

Comment:
This should check for firefox/features/sync: if it's active, show the new template, otherwise fall back.

Reply:
Good catch, fixed